src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentalDaysPrice = days * 40;
+
+  if (days <= 2) {
+    return rentalDaysPrice;
+  }
+
+  if (days > 2 && days < 7) {
+    return rentalDaysPrice - 20;
```

Comment:
What mean 20? Look like magic number. Create var

Reply:
Ok, got it